charts/incubator/tileserver-gl/questions.yaml

```diff
@@ -0,0 +1,114 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{controller}
+# Include{replicas}
+# Include{replica1}
+# Include{controllerExpertExtraArgs}
+  - variable: env
+    group: "App Configuration"
+    label: "Image Environment"
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
```

Comment:
I don't think you need this section if there's no env vars to add, might be why the catalog test fails